libr/core/cmd_cmp.c

```diff
@@ -9,14 +9,13 @@ static const char *help_msg_c[] = {
 	"c4", " [value]", "Compare a doubleword from a math expression",
 	"c8", " [value]", "Compare a quadword from a math expression",
 	"cat", " [file]", "Show contents of file (see pwd, ls)",
-	"cc", " [at] [(at)]", "Compares in two hexdump columns of block size",
-	"ccc", " [at] [(at)]", "Same as above, but only showing different lines",
-	"ccd", " [at] [(at)]", "Compares in two disasm columns of block size",
+	"cc", " [at]", "Compares in two hexdump columns of block size",
```

Comment:
this is wrong. cc* takes 2 arguments, one mandatory, 2nd or optional

Reply:
No. `cc` only parses one argument.

Reply:
uhm, right, that changed.